todo_app/app.py

```diff
@@ -1,15 +1,40 @@
-from flask import Flask
+from flask import Flask,render_template,url_for,redirect,request
+from todo_app.data import session_items
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+##@app.route('/')
+##def root():
+##    return redirect(url_for('index'))
+ 
+@app.route('/Add', methods=['POST'])
+def add_new():
+    title = request.form["title"]   
+    new_item = session_items.add_item(title)
+    return redirect(url_for('index'))
+
+##  if request.method == 'POST':
+##        new_item = session_items.add_item('Title')
+##    else:
+##        items = session_items.get_items()
+##        return ender_template('index.html',items=items)
+
+@app.route('/UpdateToDone',methods=['GET'])
```

Comment:
We did this during the call, to demonstrate how you would add a link. But I feel it's worth clarifying that in this particular case, it's bad practice. This endpoint updates stored data, and as such it shouldn't really be a "GET" endpoint.
GET implies that you are simply fetching information/loading a page, rather than modifying anything. 

So it would be better to use `<form>`s for things like this, even if there's no user input.